_docs/tasks/rate-limiting.md

```diff
@@ -83,118 +137,51 @@ Using Istio we can ensure that `1qps` is not breached.
 ## Conditional rate limits
```

Comment:
The above (step 5) sounds very wrong now:

While the load generator is running (i.e., generating more than 1 req/s), the traffic generated by your browser will be rate limited. Notice that if you log in as user "jason" or any other user, the reviews service is unable to access the ratings service, so you stop seeing stars, red or black.

Should be something like this:

While the load generator is running (i.e., generating more than 1 req/s), the traffic generated by your browser will be rate limited, unless you are logged in as user "jason". Notice that if you log in as any user other than "jason", the reviews service is unable to access the ratings service, so you stop seeing red stars. If you stop the load generator, the stars will return.